src/tests/lib/components/Popover.spec.ts

```diff
@@ -33,6 +33,22 @@ describe("Popover", () => {
       container.querySelector("div.backdrop");
 
     expect(backdrop).not.toBeNull();
+    expect(backdrop.getAttribute("class")).toContain(" visible");
```

Comment:
What do you think, could using `split` improve the test to rely on the class position/quantity?
`expect(backdrop.getAttribute("class").split(' ')).toContain(" visible");`

Reply:
I think you can say

```
e.classList.contains('foo')
```

Reply:
I think you can do

```
e.classList.contains('foo')
```

Reply:
Yes, of course, good point. Done. Thanks.